src/disProd.js

```diff
@@ -0,0 +1,128 @@
+import axios from 'axios';
+import getProducts from './getProducts';
+import CommentsPopUp from './comments';
+import Config from './config';
+
+const endPoints = new Config();
+
+const comments = new CommentsPopUp();
+
+const createElement = (el, className, source, title, id) => {
+  const elm = document.createElement(`${el}`);
+  elm.className = className;
+  if (el === 'img') {
+    elm.setAttribute('src', `${source}`);
+    elm.setAttribute('alt', 'product');
+  }
+  if (el === 'button') {
+    elm.setAttribute('type', 'submit');
+  }
+  if (el === 'h5') {
+    elm.textContent = title;
+  }
+  if (className === 'btn-comment') {
+    elm.setAttribute('data', id);
+    elm.textContent = 'Comments';
+  }
+
+  if (el === 'i') {
+    elm.setAttribute('data', id);
+  }
+
+  if (className === 'btn-reserve') {
+    elm.setAttribute('data', id);
+    elm.textContent = 'Reservations';
+  }
+  return elm;
+};
+
+const templateProduct = (source, title, id, data) => {
+  const productCard = createElement('div', 'card-product');
+  const productWrapper = createElement('div', 'card-wrapper');
+  const containerInfo = createElement('div', 'cont-info');
+  const figure = createElement('figure', 'cont-img');
+  const img = createElement('img', 'img-product', source);
+  const infoProduct = createElement('div', 'cont-info-wrapper');
+  const titleProduct = createElement('h5', 'title-product', null, title);
+  const likes = createElement('a', 'liked-cont');
+  const icon = createElement('i', 'far fa-heart', null, null, id);
+  icon.addEventListener('click', async () => {
+    let resg;
+    const body = { item_id: id };
+    try {
+      await axios.post(endPoints.likesEndPoint, body);
+      const reqg = await axios.get(endPoints.likesEndPoint);
+
+      resg = reqg;
+    } catch (err) {
+      console.log(err);
```

Comment:
- [ ] Please remember not to commit `console.log` to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website, kindly erase this and another `console.log` on your application 😄
    - This file has another `console.log` on lines: `123`
    - The `getsProducts.js` also contains these logs on lines: `12`
    - The `productCounter` also contains these logs on lines: `6`

Reply:
Resolved 🤓 . Thank you a lot for this insightful review.

Reply:
- [ ] good job on changing the suggestions the previous CR made 👍 however in the previous review was also mentioned to remove this `console.log`.